src/tape/ensemble_frame.py

```diff
@@ -259,4 +474,34 @@ def _nonempty_tapeseries(x, index=None):
 def _nonempty_tapeseries(x, index=None):
     # Construct a new TapeFrame with the same underlying data.
     df = meta_nonempty_dataframe(x)
-    return TapeFrame(df)
\ No newline at end of file
+    return TapeFrame(df)
+
+@make_meta_dispatch.register(TapeObjectFrame)
```

Comment:
I am unfamiliar with these annotations, but I'm assuming that these methods are used by dask, yeah? Not by the user?

Reply:
That is correct. We could optionally move these into a separate file for clarity.